### ISSUE TITLE
Update manifest.json

### ISSUE THREAD

**Issue by mabdelaal86** [samihaddad/vertical-tabs-chrome-extension#11]
Hi @samihaddad,

First, thanks a lot for this great extension.

I would like to contribute to the extension. Specially, I want to implement this one: https://github.com/samihaddad/vertical-tabs-chrome-extension/issues/6

I tried to load the extension source code into Chrome, but I faced the following issue:
```
Manifest version 2 is deprecated, and support will be removed in 2023
```

Should I update the manifest structure into version 3?

I have also noticed that the latest version on Chrome webstore "2.1.7" is different than the one in source code.
I'm wondering if you update the extension on Chrome webstore using a different code base!

**Comment by samihaddad** [comment]
Hey @mabdelaal86! Indeed the source code is a bit old, as I rewrote it from scratch in version 2 and haven't prepared it for public use yet. Appreciate your interest to contribute, but I've already implemented #6 and will release it this week. Will start preparing it for contribution and push the latest code soon when it's ready and you can let me know if you'd like to contribute to anything else in the future!

**Comment by mabdelaal86** [comment]
Glad to hear that you have already implemented #6. I don't have any specific idea in mind to contribute, but I'm looking forward to helping improve the extension.
